examples/tabs/js/AltManager.js

```diff
@@ -0,0 +1,58 @@
+var Alt = require('../../../src/alt');
+
+class AltManager {
+
+  constructor() {
+    this.alts = {};
+  }
+
+  create(altKey) {
+    if (this.get(altKey)) {
+      throw new Error(`Alt key ${altKey} already exists`);
+    }
+
+    this.alts[altKey] = new Alt();
+    return this.alts[altKey];
+  }
+
+  get(altKey) {
+    return this.alts[altKey];
+  }
+
+  // returns all alt instances
+  all() {
+    return this.alts;
+  }
+
+  findWhere(regex) {
+    var results = [];
+    for (var i in this.alts) {
+      if (regex.exec(i) === null) {
+        continue;
+      }
+
+      results[i] = this.alts[i];
+    }
+
+    return results;
+  }
+
+  delete(altKey) {
+    if (!this.get(altKey)) {
+      throw new Error(`Alt key ${altKey} does not exist`);
+    }
+
+    delete this.alts[altKey];
+  }
+
+  createOrGet(altKey) {
```

Comment:
looks like you try to get first and then create so you should rename this method accordingly.
``` getOrCreate ```